src/styles/blocks/_about.scss

```diff
@@ -0,0 +1,75 @@
+.about {
+
+  padding: 18.75vw 6.25vw;
```

Comment:
Use pixels or rems instead of relative viewport units. It causes a lot of problems with your layout.
Fix it everywhere.